WPMedia/arc-themes-components
# THEMES-1128: Fix null value in SASS Mixin

## Ticket

- [THEMES-1129](https://arcpublishing.atlassian.net/browse/THEMES-1129)

## Description

Resolve a null token issue with the global Sass mixins when a breakpoint is added without a style to go with it.

## Acceptance Criteria

Sass Mixin should not have an issue with breakpoints that have no styles.

## Test Steps

1. Checkout branch - `git checkout themes-1129`
2. Update dependencies - `npm i`
3. Run Local stylebuilder (don't make me put the instructions here!) (Link the `arc-themes-components`)
4. Add a new breakpoint to the `themes/styles/breakpoints.json` file.
5. The local fusion build should not through a webpack error when the Mixin finds a breakpoint with no styles.

## Author Checklist


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1129]: https://arcpublishing.atlassian.net/browse/THEMES-1129?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
